phpstan.neon.dist

```diff
@@ -0,0 +1,13 @@
+includes:
+    - 'vendor/phpstan/phpstan-phpunit/extension.neon'
+    - 'vendor/phpstan/phpstan-phpunit/rules.neon'
+
+parameters:
+    paths:
+        - 'src'
+        - 'tests'
+    level: 'max'
+    checkGenericClassInNonGenericObjectType: false
+    ignoreErrors:
+        - '#Cannot call method integerNode\(\) on Symfony\\Component\\Config\\Definition\\Builder\\NodeParentInterface\|null\.#'
+        - '#Parameter \#1 \$argument of class ReflectionClass constructor expects class-string\<object\>\|object\, object\|string\|null given\.#'
```

Comment:
it seems to be "easily" fixable by using a variable with a non-nullable type

maybe something like: 
```php
        $controllerAttributeParts = explode('::', $controllerAttribute);
        $controllerAttributePart1 = $controllerAttributeParts[0] ?? '';
        $controllerAttributePart2 = $controllerAttributeParts[1] ?? '';

        if (!class_exists($controllerAttributePart1)) {
            // If controller is passed with alias until class name
            $serviceIdAttribteParts = explode(':', $controllerAttributePart1);
            if (!$this->container->has($serviceIdAttribteParts[0])) {
                throw new \InvalidArgumentException('Parameter _controller from request : "'.$controllerAttribute.'" do not contains a valid class name');
            }
            $controllerAttributePart1 = $this->container->get($serviceIdAttribteParts[0]) ?? '';
            $controllerAttributePart2 = $serviceIdAttribteParts[1] ?? '';
        }
        $reflection = new \ReflectionClass($controllerAttributePart1);
...
```